engine/gamesys/src/gamesys/gamesys_private.cpp

```diff
@@ -59,7 +59,20 @@ namespace dmGameSystem
         dmLogError("%s", buf);
 
         va_end(lst);
+    }
 
+    void ShowFullBufferError(const char* object_name, const char* config_key, int max_count)
+    {
+        if (config_key)
+        {
+            dmLogError("%s could not be created since the buffer is full (%d). Increase the '%s' value in game.project",
```

Comment:
Perhaps add link to game.project as well?

https://github.com/defold/defold/issues/5252#issuecomment-702072990